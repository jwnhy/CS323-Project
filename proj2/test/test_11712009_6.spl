int main(int a, int b) {
    1 = a;
}