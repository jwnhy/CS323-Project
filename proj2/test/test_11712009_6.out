Error type 6 at Line 2: LHS of assignment is not a left value.
